alembic/env.py

```
@@ -5,19 +5,21 @@
 This template provides the basic async setup for PostgreSQL migrations.
 
 """
+
 import asyncio
 from logging.config import fileConfig
+
 from sqlalchemy import engine_from_config, pool
 from sqlalchemy.ext.asyncio import AsyncEngine
-from alembic import context
 
+from alembic import contex
```

Flake8 (7.2.0)

[error] 'alembic.contex' imported but unused

(F401)